grpcclient/lc_client.go

```diff
@@ -166,14 +167,20 @@ func NewLcClient(setters ...LcClientOption) *LcClient {
 		setter(cli)
 	}
 
+	retryOpts := []grpc_retry.CallOption{
```

Comment:
can you make it configurable ? num of retries and backoff ?

Reply:
there is already a setter: https://github.com/vchain-us/ledger-compliance-go/blob/dadda9f4b5514c85f2fed4f78eaa332b58ac89a1/grpcclient/lc_client_options.go#L61 
`func DialOptions(dopts []grpc.DialOption) LcClientOption {
	return func(args *LcClient) {
		args.DialOptions = dopts
	}
}`

so seems that it is already configurable. 

Overall, it must be configurable so we can easily tune it without releasing vcn :)

Reply:
summarizing, make it configurable either way :)

Reply:
it's ok for me to refactor it.

Reply:
added configuration option